tornjak-frontend/src/components/spiffe-helper.tsx

```diff
@@ -187,7 +195,7 @@ class SpiffeHelper extends Component {
 
   // numberEntriesOfAgent takes in an agent and list of entries
   // returns number of entries in the agent
-  numberEntriesOfAgent(agent, globalEntries) {
+  numberEntriesOfAgent(agent: AgentsList, globalEntries: EntriesList[] | undefined) {
```

Comment:
this should not expect something that is undefined. check should be on the caller if its just going to check undefined and return.

Reply:
Done!